### PR TITLE
fix kubectl and storage typo

### PR DESCRIPTION
**Pull request by panguicai008**
#### What this PR does / why we need it:
fix kubectl and storage typo in the document
#### Does this PR introduce a user-facing change?
 NONE

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
pick nit

File: docs/i18n/cn/docusaurus-plugin-content-docs/current/architecture/modules/ldm.md [hunk 1]
```diff
@@ -39,10 +39,10 @@ sidebar_label: "本地磁盘管理器"
     该命令用于获取集群中磁盘资源信息，获取的信息总共有四列，含义分别如下：
 
     - **NAME：** 代表磁盘在集群中的名称。
-    - **NODEMATCH：** 表明磁盘所在的节点名称。 
+    - **NODEMATCH：** 表明磁盘所在的节点名称。
     - **PHASE：** 表明这个磁盘当前的状态。
 
-    通过`kuebctl get localdisk <name> -o yaml` 查看更多关于某块磁盘的信息。
+    通过`kubectl get localdisk <name> -o yaml` 查看更多关于某块磁盘的信息。
```
[the remark this answers is not in the document]
Comment (panguicai008) [reply docs/i18n/cn/docusaurus-plugin-content-docs/current/architecture/modules/ldm.md:45]: OK，thanks

[the remark this answers is not in the document]
Comment (panguicai008) [reply docs/i18n/cn/docusaurus-plugin-content-docs/current/architecture/modules/ldm.md]: I have updated it

**Comment by windsonsea** [comment]
Thanks. LGTM

**Pull request merged**